word_list.go

```diff
@@ -3,17 +3,17 @@ package mnemonicode
 // WordListVersion is the version of compiled in word list.
 const WordListVersion = "0.7"
 
-var wordMap = make(map[string]int, len(wordList))
+var wordMap = make(map[string]int, len(WordList))
 
 func init() {
-	for i, w := range wordList {
+	for i, w := range WordList {
 		wordMap[w] = i
 	}
 }
 
 const longestWord = 7
 
-var wordList = []string{
+var WordList = []string{
```

Comment:
Adding a documentation comment didn't seem necessary as anything I wrote would've likely been something tautological such as "WordList is the word list."